README.md

```diff
@@ -12,29 +12,29 @@ npm install --save quicktube
 
 ### Initial setup
 
-Is really simple. Just add the video ID to `data-quicktube-video='{video-id}'`
+Is really simple. Just add the video ID to `data-quicktube='{video-id}'` and `data-quicktube-play='{video-id}'` to the play trigger element.
 
 ```html
-<div class="quicktube" data-quicktube="kittens">
-    <div class="quicktube__video" data-quicktube-video="k6QanQUaDOo">
-    </div>
-    <div data-quicktube-play="kittens" class="quicktube__poster" data-quicktube-poster>
-        <div class="quicktube__play quicktube__btn">
+<div class="quicktube" data-quicktube="k6QanQUaDOo">
+    <div class="quicktube__video" data-quicktube-video></div>
+    <div data-quicktube-play="k6QanQUaDOo" class="quicktube__poster" data-quicktube-poster>
+        <!-- Optional poster frame image -->
+        <img class="quicktube__poster-image" src="/path/to/image" />
+        <button class="quicktube__play quicktube__btn">
             Play
-        </div>
+        </button>
     </div>
 </div>
 ```
 
-Initialise Quicktube once jQuery is ready.
+Initialise quicktube.
 
-```js
+```javascript
```

Comment:
Haha, think there were some conflicts in this file which were tricky to figure out. I imagine this just snuck through